.travis.yml

```diff
@@ -4,9 +4,17 @@ node_js:
 - node
 - 6
 - 4
-before_install: yarn global add greenkeeper-lockfile@1
+before_install:
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
+- export PATH="$HOME/.yarn/bin:$PATH"
+- yarn global add greenkeeper-lockfile@1
 before_script: greenkeeper-lockfile-update
+script:
+- yarn commitlint --from="$TRAVIS_BRANCH" --to="$TRAVIS_COMMIT"
+- yarn commitlint --from=$TRAVIS_COMMIT
+- yarn test
 after_script: greenkeeper-lockfile-upload
+after_success: yarn semantic-release
```

Comment:
Can we skip node 4 somehow? If not, let’s drop it and release next version as a major

Reply:
Could add it as a dependency in the build stages.